### DISCUSSION TITLE
Opening Search Box when typing in Windows 10 Startmenu

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#990] [Ideas]
Hi :) 

Thank you for this amazing app, wondering if it's possible if the Windows 10 Start menu when opened and typing words, it will open the search box?

Right now it does seem to tell the search box the words I type but I have to either Win + S or open the search box to press enter to open an app or search something.

Is there a possible fix?

Thanks,

Shawn

**Comment by Poopooracoocoo** [comment]
(a fix would be great if possible. For me it works _sometimes_. It's defs a bug with the Windows 10 start menu - and windows 10 made me get in the habit of pressing win+s)

**Reply by ghost** [reply]
It seems according to the owner of the tool in the discussions on another one that it isn`t possible. Anyways Win + S works for me :) 

have a wonderful ay!

shawn

**Reply by valinet** [reply]
Where did I seem it seems so? For me, it works 100% of the times. I press Start, Windows 10 menu pops up, start typing, the search box appears and takes in the words I typed. Windows 11 build 22000.493. Again I say, tested on that, can only speak for that.

**Reply by Poopooracoocoo** [reply]
My bad. Looks like Microsoft fixed this in the insider builds between Windows 10 and 11 because I've searched on the start menu fine a few times now. I still try to press Win+S though. Old habits die hard.

**Reply by adnan1710** [reply]
Search works fine on Windows 11 build 22000.527. I just open start menu and start typing.
